tests/appsec/iast/_test_iast_fixtures.py

```diff
@@ -129,6 +129,12 @@ def test_insecure(self):
             expected_evidence=self.expected_evidence,
         )
 
+    def check_test_insecure(self):
+        # to avoid false positive, we need to check that iast is implemented
+        # AND that the secure endpoint is not vulnerable
```

Comment:
I don't understand....."check_test_insecure --> AND that the secure endpoint is not vulnerable" ¿??

Reply:
failure on copy and paste from previous code, this comment was from line 152, in `test_secure` method. I'll change the comment.